api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,22 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Global target percentage of requests that will be force traced if the
+    // x-client-trace-id header is set. Must be an integer number between 0 and
```

Comment:
The convention we have for headers in docs is `*x-client-trace-id*`.

Reply:
Fixed.